userbot/events.py

```diff
@@ -51,6 +56,11 @@ def register(**args):
             args['pattern'] = pattern.replace('^.', unsafe_pattern, 1)
 
     def decorator(func):
+        try:
```

Comment:
WTH! That is seriously the wrong way to go about this!

Reply:
Then tell me how

Reply:
Suggest another method

Reply:
You can import the module to a temporary namespace and unload each function it exports